eng/targets/Helix.targets

```diff
@@ -16,10 +16,6 @@
     <HelixPreCommand Include="call RunPowershell.cmd InstallNode.ps1 $(NodeVersion) %25HELIX_CORRELATION_PAYLOAD%25\node\bin || exit /b 1" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(IsHelixJob)' == 'true' AND '$(TestDependsOnAspNetRef)' == 'true' AND '$(IsTargetingPackBuilding)' == 'true'">
-    <HelixContent Include="$(RepoRoot)artifacts\packages\Release\Shipping\Microsoft.AspNetCore.App.Ref.$(AppRuntimeVersion).nupkg" />
-  </ItemGroup>
-
   <ItemGroup Condition="'$(IsHelixJob)' == 'true' AND '$(TestDependsOnAspNetRuntime)' == 'true'">
     <HelixContent Include="$(RepoRoot)artifacts\packages\Release\Shipping\*-ci.nupkg" />
```

Comment:
What needs every package we ship❔ And, does including these packages in the correlation payload mean they're available when restoring packages❔ This isn't reliable in any case because it's often evaluated before projects build or publish.

Reply:
The template tests use a wide variety of packages we build, rather than listing them all, it seems reasonable for us to just install the ones we build, these aren't in the correlation payload, they show up in the workitem root

Reply:
Ah,
``` text
nuget add source {Options.HELIX_WORKITEM_ROOT} --configfile NuGet.config 
```